packages/vite/src/node/server/middlewares/indexHtml.ts

```diff
@@ -372,7 +376,20 @@ const devHtmlHook: IndexHtmlTransformHook = async (
               config.logger,
             )
           }
-          content = getCodeWithSourcemap('css', result.code, result.map)
+          content = getCodeWithSourcemap(
+            lang as
+              | 'js'
+              | 'css'
+              | 'less'
+              | 'sass'
+              | 'scss'
+              | 'styl'
+              | 'stylus'
+              | 'pcss'
+              | 'postcss',
```

Comment:
Can we simplify this as `lang === 'js' ? 'js' : 'css'` instead? This function injects the sourcemap comment, and there's only two injection styles (js/css) that it should support.

Reply:
@bluwy made the changes.thanks